pkg/config/config.go

```diff
@@ -1305,3 +1310,29 @@ func (e eventsLogMaxSize) MarshalText() ([]byte, error) {
 	}
 	return []byte(fmt.Sprintf("%d", e)), nil
 }
+
+func LoggingValid(logDriver string, eventsLogger string) error {
+
+	validLogDriver := []string{"file", "k8s-file", "journald", "none"}
+	validEventsLogger := []string{"file", "journald", "none"}
+	if !util.StringInSlice(logDriver, validLogDriver) {
+		return errors.Errorf("invalid LogDriver %q must be one of %q", logDriver, strings.Join(validLogDriver, ", "))
+	}
+	if !util.StringInSlice(eventsLogger, validEventsLogger) {
+		return errors.Errorf("invalid EventsLogger %q must be one of %q", eventsLogger, strings.Join(validEventsLogger, ", "))
+	}
+	logErr := errors.Errorf("invalid logging combination LogDriver %q is not allowed with EventsLogger %q", logDriver, eventsLogger)
```

Comment:
Please use `fmt.Errorf`.

Reply:
Perhaps it is best to change to a Warnf and be done with it.

Reply:
Can you elaborate on why we need this check?  It seems legitimate to me to not use the same backend for logs and events but I may be missing something.

Reply:
Full context is: https://github.com/containers/podman/issues/12175 (podman-remote, different log drivers).

Reply:
Bottom line this is not something we recommend and we should warn against it.  I agree that we can not error on this.